src/Property.ts

```diff
@@ -147,7 +147,7 @@ export default class Property<T = any> {
 		return this.getString().replace(/[0-9\.\-]/g, '');
 	}
 
-	getPixels(axis?: Axis, processPercent = false) {
+	getPixels(axis?: Axis, processPercent = false, considerParentFontSize = false) {
```

Comment:
@dangreen i took this from https://github.com/canvg/canvg/pull/749 ... we can also choose to use named options here to not have to pass in double `undefined`.

Reply:
@gabelerner 

```ts
getPixels(axis?: Axis, processPercent?: boolean);
getPixels(isFontSize?: boolean);
getPixels(axisOrIsFontSize?: Axis | boolean, processPercent = false) {

  const [
    axis,
    isFontSize
  ] = typeof axisOrIsFontSize === 'boolean'
    ? [undefined, axisOrIsFontSize]
    : [axisOrIsFontSize];
  ...
    case this.isString(/%$/) && isFontSize:
      return this.getNumber() * this.getEm(/* viewPort */);
   
    case this.isString(/%$/):
      this.getNumber() * viewPort.computeSize(axis);
  ...
}
```